chirp/drivers/baofeng_uv17.py

```diff
@@ -43,7 +43,6 @@ def _get_memory_map(radio):
         frame = radio._make_frame(b"R", addr, 1)
         baofeng_common._rawsend(radio, frame)
         blocknr = ord(baofeng_common._rawrecv(radio, 6)[5:])
```

Comment:
I saw this while I was looking at this driver for that bug. If `_rawrecv()` returns a `bytes` then you just need this:
```
baofeng_common._rawrecv(radio, 6)[5]
```
Which will be an int. Example:
```
>>> b = b'\x01\x02\x03'
>>> b[1]
2
```
Maybe clean that up while you're here?

Reply:
True... What was I doing there.. ? :-)